FHICORC.Backend/FHICORC.Tests.UnitTests/DGCGComponentTests/DGCGRevocationServiceComponentTests.cs

```diff
@@ -63,6 +63,58 @@ public void Setup()
             }
         }
 
+        private Tuple<List<string>, List<string>> GetRandomHashesInLists()
```

Comment:
Update to private (ICollection<string> HashCollectionOne, ICollection<string> HashCollectionTwo)